<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1709</width>
    <height>1051</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Osdag</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget::showMaximised()</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_help">
      <property name="styleSheet">
       <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}</string>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_openfile">
      <property name="styleSheet">
       <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}</string>
      </property>
      <property name="text">
       <string>Open file</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QStackedWidget" name="myStackedWidget">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Osdagpage">
       <widget class="QLabel" name="lbl_OsdagHeader">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>1871</width>
          <height>381</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ResourceFiles/osdag_icons.qrc">:/newPrefix/images/Osdag_header.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_fosseelogo">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>670</y>
          <width>250</width>
          <height>92</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ResourceFiles/osdag_icons.qrc">:/newPrefix/images/Fossee_logo.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_iitblogo">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>590</y>
          <width>290</width>
          <height>201</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ResourceFiles/osdag_icons.qrc">:/newPrefix/images/iit_logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="connectionpage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="mytabWidget">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;#&quot;&gt;Shear Connection&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QTabBar::tab {
    margin-right: 10;
 }
QTabBar::tab::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QTabBar::tab{
height: 40px;
width: 200px;
background-color: #925a5b;
color:#ffffff;
}
QTabBar::tab{
border-top-left-radius: 2px ;
border-top-right-radius: 2px ;
border-bottom-left-radius: 0px ;
border-bottom-right-radius: 0px ;
}
 </string>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab1_shearconnection">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <attribute name="title">
            <string>Shear Connection</string>
           </attribute>
           <widget class="QRadioButton" name="rdbtn_finplate">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>30</y>
              <width>131</width>
              <height>27</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Fin Plate</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="rdbtn_seated">
            <property name="geometry">
             <rect>
              <x>830</x>
              <y>330</y>
              <width>101</width>
              <height>27</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Seated</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="rdbtn_cleatangle">
            <property name="geometry">
             <rect>
              <x>830</x>
              <y>30</y>
              <width>151</width>
              <height>27</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton {
text-shadow : black 0.1em 0.1em 0.2em  ;
}</string>
            </property>
            <property name="text">
             <string>Cleat Angle</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="rdbtn_endplate">
            <property name="geometry">
             <rect>
              <x>340</x>
              <y>330</y>
              <width>131</width>
              <height>27</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>End Plate</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_start">
            <property name="geometry">
             <rect>
              <x>620</x>
              <y>620</y>
              <width>200</width>
              <height>32</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="lbl_cleat">
            <property name="geometry">
             <rect>
              <x>860</x>
              <y>80</y>
              <width>200</width>
              <height>200</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ResourceFiles/osdag_icons.qrc">:/newPrefix/images/cleatangle.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="lbl_endplate">
            <property name="geometry">
             <rect>
              <x>370</x>
              <y>380</y>
              <width>200</width>
              <height>171</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ResourceFiles/osdag_icons.qrc">:/newPrefix/images/endplate.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="lbl_seat">
            <property name="geometry">
             <rect>
              <x>860</x>
              <y>380</y>
              <width>200</width>
              <height>200</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ResourceFiles/osdag_icons.qrc">:/newPrefix/images/seated.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="lbl_finplate">
            <property name="geometry">
             <rect>
              <x>360</x>
              <y>80</y>
              <width>201</width>
              <height>161</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ResourceFiles/osdag_icons.qrc">:/newPrefix/images/finplate.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab2_momentconnection">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <attribute name="title">
            <string>Moment Connection</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tensionpage">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>230</y>
          <width>271</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>19</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Coming Soon.....</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="compressionpage"/>
      <widget class="QWidget" name="flexuralpage"/>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="myListWidget">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>84</green>
            <blue>69</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>84</green>
            <blue>69</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>194</green>
            <blue>80</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget
{
background-color: #abc250 ;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
      <item>
       <property name="text">
        <string> Design :</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </item>
     </widget>
    </item>
   </layout>
   <widget class="QPushButton" name="btn_connection">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}
</string>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tension">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}
</string>
    </property>
    <property name="text">
     <string>Tension Member</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_compression">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}
</string>
    </property>
    <property name="text">
     <string>Compression Member</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_flexural">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}</string>
    </property>
    <property name="text">
     <string>Flexural Member</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_beamCol">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}</string>
    </property>
    <property name="text">
     <string>Beam-Column</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}</string>
    </property>
    <property name="text">
     <string>Plate Girder</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gantry">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::hover
{
   background-color: #d97f7f;
   color:#000000 ;
}

QPushButton
{
background-color: #925a5b;
color:#ffffff;
}</string>
    </property>
    <property name="text">
     <string>Gantry Girder</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>myListWidget</zorder>
   <zorder>myStackedWidget</zorder>
   <zorder>btn_openfile</zorder>
   <zorder>btn_help</zorder>
   <zorder>btn_connection</zorder>
   <zorder>btn_tension</zorder>
   <zorder>btn_compression</zorder>
   <zorder>btn_flexural</zorder>
   <zorder>btn_beamCol</zorder>
   <zorder>btn_plate</zorder>
   <zorder>btn_gantry</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1709</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="ResourceFiles/osdag_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_connection</sender>
   <signal>clicked()</signal>
   <receiver>mytabWidget</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>985</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
